ci-operator/jobs/openshift-psap/special-resource-operator/openshift-psap-special-resource-operator-release-4.11-periodics.yaml

```diff
@@ -13,6 +13,16 @@ periodics:
     ci.openshift.io/generator: prowgen
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
   name: periodic-ci-openshift-psap-special-resource-operator-release-4.11-e2e-aws-ocp-4-8-periodic
+  reporter_config:
```

Comment:
Unrelated to OKD, but I'm fine keeping it

Reply:
Yeah, sorry for these - manual modifications are not continuously synced with master, so my procedure for syncing these fixes up all jobs that got out of sync over time. Let me separate these to another PR.